otwcode/otwarchive
# AO3-6297: Stop using MimeMagic in downloads code

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6297

## Purpose

Replaces MimeMagic with Marcel

## Testing Instructions
Log in

Post > New Work (we'll want a fresh work so we're sure we're not getting cached files)

Fill in required information

Post

Expand the "Downloads" menu on the work

Verify you can download and open each type of file

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

warlockmel (she/her)
